docs/release-notes/14-0-0/README.md

```diff
@@ -0,0 +1,214 @@
+- Feature: Allow setting createdAt, author via API \[[#53423](https://community.openproject.org/wp/53423)\]
+- Feature: Make the Author field editable or settable \[[#53444](https://community.openproject.org/wp/53444)\]
+- Feature: Allow 4 and 8 week display modes in team planner \[[#53475](https://community.openproject.org/wp/53475)\]
+- Feature: Allow umlauts for login name in OpenProject, LDAP
+  authentication \[[#53486](https://community.openproject.org/wp/53486)\]
+- Feature: Meetings: Improve attachments \[[#53506](https://community.openproject.org/wp/53506)\]
+- Feature: Support setting accountable to current user via custom
+  action \[[#53507](https://community.openproject.org/wp/53507)\]
+- Feature: Add separate checkbox about attachments when user copies a
+  meeting \[[#53568](https://community.openproject.org/wp/53568)\]
+- Feature: Handle no active project attributes in project overview
+  sidebar \[[#53577](https://community.openproject.org/wp/53577)\]
+- Feature: Check if and how project activity logs should be
+  adapted \[[#53580](https://community.openproject.org/wp/53580)\]
+- Feature: Disable the macros for the project custom fields. \[[#53701](https://community.openproject.org/wp/53701)\]
+- Feature: Project creation with project attributes \[[#53703](https://community.openproject.org/wp/53703)\]
+- Feature: Project copy with project attributes \[[#53705](https://community.openproject.org/wp/53705)\]
+- Feature: Project export with disabled project attributes \[[#53733](https://community.openproject.org/wp/53733)\]
+- Feature: Removing a work package from a agenda should be called "Remove" instead of "
+  Delete" \[[#53766](https://community.openproject.org/wp/53766)\]
+- Feature: Add % Complete to section "Estimates and time" \[[#53771](https://community.openproject.org/wp/53771)\]
+- Feature: Show diff for changes in meeting agenda items \[[#53975](https://community.openproject.org/wp/53975)\]
+- Feature: Progress reporting for work package hierarchies \[[#40867](https://community.openproject.org/wp/40867)\]
+- Feature: Custom set of project attributes grouped in sections \[[#49688](https://community.openproject.org/wp/49688)\]
+- Feature: Copy of template projects including their project folders in
+  SharePoint \[[#51000](https://community.openproject.org/wp/51000)\]
+- Feature: Email notifications for unhealthy file storages \[[#52840](https://community.openproject.org/wp/52840)\]
+
+<!-- END AUTOMATED SECTION -->
+<!-- Warning: Anything above this line will be automatically removed by the release script -->
+
+#### Contributions
+
+A big thanks to community members for reporting bugs and helping us identifying and providing fixes.
+
+Special thanks for reporting and finding bugs go to
+
+Silas Kropf, Philipp Schulz, Benjamin Rönnau, Mario Haustein, Matt User, Mario Zeppin, Romain Besson, Cécile Guiot,
+Daniel Hilbrand, Christina Vechkanova, Sven Kunze, Richard Richter, Julian Wolff
```

Comment:
This file seems to be misplayed in the PR.

Reply:
This is by design, so we can add the breaking change warning